### ISSUE TITLE
about trained weights

### ISSUE THREAD

**Issue by Brizel** [eriklindernoren/PyTorch-YOLOv3#28]
I followed the instruction and trained a model on COCO dataset which is provided  by the author. After 20 epochs ,the total loss is about 0.2-0.3. When I use this new trained weights to detect the sample images, no detection can be made. Does anybody have the same situation?

**Comment by strive1127** [comment]
Core was generated by `python train.py'. Program terminated with signal SIGSEGV, Segmentation fault.
Core dumped!!!

**Comment by eriklindernoren** [comment]
The loss decreases for me as well and the recall and precision seems to increase but when loading the weights and setting `model.eval()` the model does not give reasonable outputs. Not sure why this is but I will try to investigate it as soon as I have the time.

**Comment by Brizel** [comment]
Thanks for your reply.
I also tried this trained weights on the original C code, but no reasonable outputs can be given either. Maybe the problem is somewhere in training codes.

**Comment by TanMengluoye** [comment]
Thanks.
Did you solve the problem?I have the same problem.
I try training with my own data,but the saved model does not give good reasonable outputs.

**Comment by Brizel** [comment]
I'm sorry that I haven't solve this problem yet.
